Error importing "github.com/aws/aws-sdk-go/aws"

After executing
```shell
GO111MODULE=off go get -v "github.com/aws/aws-sdk-go/aws"
```
the following program `sample.go` triggers a panic:

```go
package main

import "github.com/aws/aws-sdk-go/aws"

func main() {
  println(*aws.String("Hello from aws!"))
}
```

Expected result:
```console
$ GO111MODULE=off go run ./sample.go
Hello from aws!
```

Got:
```console
$ yaegi ./sample.go
run: ./sample.go:3:8: import "github.com/aws/aws-sdk-go/aws" error: /home/max/go/src/github.com/aws/aws-sdk-go/aws/config.go:7:2: import "github.com/aws/aws-sdk-go/aws/credentials" error: /home/max/go/src/github.com/aws/aws-sdk-go/aws/credentials/chain_provider.go:4:2: import "github.com/aws/aws-sdk-go/aws/awserr" error: cannot use  (type interfaceT) as type errorT in return argument
```

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -1433,7 +1433,7 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 				// TODO(mpl): move any of that code to typecheck?
 				c.typ.node = c
 				if !c.typ.assignableTo(typ) {
-					err = fmt.Errorf("cannot use %v (type %v) as type %v in return argument", c.ident, c.typ.cat, typ.cat)
+					err = c.cfgErrorf("cannot use %v (type %v) as type %v in return argument", c.ident, c.typ.cat, typ.cat)
 					return
 				}
 				if c.typ.cat == nilT {

--- a/interp/type.go
+++ b/interp/type.go
@@ -1065,6 +1065,10 @@ func (t *itype) methods() methodSet {
 		seen[typ] = true
 
 		switch typ.cat {
+		case aliasT:
+			for k, v := range getMethods(typ.val) {
+				res[k] = v
+			}
 		case interfaceT:
 			// Get methods from recursive analysis of interface fields.
 			for _, f := range typ.field {

